python/lsst/obs/subaru/isr.py

```diff
@@ -307,24 +337,23 @@ def runDataRef(self, sensorRef):
             self.darkCorrection(ccdExposure, darkExposure)
         if self.config.doFlat:
             flatExposure = self.getIsrExposure(sensorRef, "flat")
-            self.flatCorrection(ccdExposure, flatExposure)
+            self.flatCorrection(ccdExposure, flatExposure, doTweakFlat=self.config.doTweakFlat)
 
-        if self.config.doDefect:
+        if self.config.doDefect and not interpolationDone:
             self.maskAndInterpDefect(ccdExposure, defects)
```

Comment:
Why leave this here?

Reply:
Where would I move it instead?  I think it's the right place to do defect interpolation if you're not running brighter-fatter (or at least one of several right places, and it's the old one).

Reply:
It took me a while to figure out that `not interpolationDone` effectively means `not self.config.doBrighterFatter`. In that case, I think you're right.